crates/next-core/src/next_client/context.rs

```diff
@@ -140,7 +140,7 @@ pub async fn get_client_compile_time_info(
 }
 
 #[turbo_tasks::value(shared, serialization = "auto_for_input")]
```

Comment:
We should be able to remove this then (and probably derive a few more traits instead)
```suggestion
```

Reply:
ClientContextType is stored as a task output here:

https://github.com/vercel/next.js/blob/895908a35eb376cd5feb828ad34b8155a0caf0f1/crates/next-api/src/app.rs#L156

so i think it needs to be a value?

Or do you think i should just make that not be a turbotask fn? it does seem trivial